src/types/ErrorType.ts

```diff
@@ -0,0 +1,6 @@
+export enum ErrorType {
+  none,
+  postsLoadingError,
+  commentsLoadingError,
+  deleteCommentError,
+}
```

Comment:
```suggestion
export enum ErrorType {
  None,
  PostsLoadingError,
  CommentsLoadingError,
  DeleteCommentError,
}
```
use PascalCase for enum keys